src/scripts/helpers/utils.js

```diff
@@ -5,7 +5,7 @@ export function float_2(d) {
 export function arrays_equal(arr1, arr2) {
 	if(arr1.length !== arr2.length) return false;
 	let are_equal = true;
-	arr1.map((d, i) => {
+	arr1.forEach((d, i) => {
```

Comment:
would require a bit of change in the inner block but using [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) is better since it is equivalent to `for` loop with `break`

Reply:
Could you pass a more specific example.

Reply:
`let are_equal = !arr1.some((item, index) => item !== arr2[index])` would do the same thing but with a better performance, since it would stop the loop once an item in `arr1` is not equal to the item at same index of `arr2`. So `some`solution is like using:

```typescript
let are_equal = true;
for (let i = 0; i < arr1.length; i++) {
  if (arr1[i] !== arr2[i]) {
    are_equal = false;
    break;
  }
}
```

However currently it checks all elements no matter which is redundant

Reply:
Done.
Btw, your code example above does not use ```some``` :smile: